### PR TITLE
Add USDT token address in Mode chain

### PR DESCRIPTION
**Pull request by ssallam**
This is the main USDT token on Mode network to be used with the StandardBridge and is paired with the USDT token `0xdac17f958d2ee523a2206206994597c13d831ec7` on Ethereum mainnet.

USDT token address on Mode is `0xf0F161fDA2712DB8b566946122a5af183995e2eD` and is referenced in Mode's main docs https://docs.mode.network/mode-developer-mainnet/mainnet-contract-addresses#token-addresses

This token is deployed using OptimismMintableERC20 directly without using the factory because the factory only supports 18 decimals while USDT requires 6 decimals.

### CODE REVIEW

**Comment by hamdiallam** [comment]
Hey @ssallam, mode needs to be added to the `chains.ts` file so that the checks can appropriately be run

**Comment by ssallam** [comment]
> chains.ts

Thanks, added Mode to chains.ts

**Comment by ssallam** [comment]
[hamdiallam](https://github.com/hamdiallam)
Can you please merge these PRs as well:
https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/578
https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/580
https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/581
https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/582
https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/583
https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/584
https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/585
https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/586

**Comment by ssallam** [comment]
[hamdiallam](https://github.com/hamdiallam) anything blocking this PR?

**Comment by hamdiallam** [comment]
Hey @ssallam,

Like my comments in the other PRs, can you provide USDT documentation that this is the official contract address to be used on the Mode network?

Given the 1:many relationship between L1->L2 tokens, using the factory to just create the tokens isn't sufficient as each project may choose to create a canonical version themselves

**Comment by agileurbanite** [comment]
@hamdiallam can you provide some clarification on what documentation is needed? looking at a previous PR we don't see documentation link so would appreciate clarity: https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/527

**Comment by agileurbanite** [comment]
@hamdiallam this is the contract address on L1:
https://etherscan.io/token/0xdac17f958d2ee523a2206206994597c13d831ec7

A little about mode as part of OP superchain here:
https://twitter.com/Optimism/status/1749872693451817188

**Comment by agileurbanite** [comment]
@hamdiallam any updates here?

**Comment by ssallam** [comment]
Hi @hamdiallam
I added information about this token in the top comment/description. You can see this token address in Mode's main docs (link above)

Can you please re-open/merge this PR, appreciated.

**Comment by ssallam** [comment]
https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/599

**Comment by ssallam** [comment]
@roberto-bayardo @cfluke-cb can you please help get this PR approved?

**Comment by roninjin10** [comment]
@ssallam this is gonna be blocked for a little bit longer because mode is not yet supported by this list

**Comment by agileurbanite** [comment]
@roninjin10 we combined that change into this PR as you can see here to get the chain added: 
image
 should it be a separate PR?

**Comment by hamdiallam** [comment]
> @roninjin10 we combined that change into this PR as you can see here to get the chain added: image should it be a separate PR?

It'd be useful to have it be a separate PR. Little bit blocked atm to get in to define the token policy for the Mode network before we can start merging in some of these PRs that add tokens

**Comment by roninjin10** [comment]
Will try to get this pr unblocked this week

**Review by hamdiallam — APPROVED** [review]

**Comment by hamdiallam** [comment]
Confirmed with the Mode team that they want to take responsibility for ownership of canonical token address representation vs the respective teams

**Pull request merged**
